PractiseJuly7/EmployeeFilter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseJuly7
+{
+    public class EmployeeFilter
```

Comment:
That was not a home task, but if you fulfilled that by yourself - that is really good.